### PR TITLE
docs: update CVEs fixed on 3.0.2 and 2.1.3

### PR DESCRIPTION
**Pull request by dpgaspar**
### SUMMARY

Update CVEs fixed on 3.0.2 and 2.1.3 on our documentation

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: docs/docs/security/cves.mdx [hunk 1]
```diff
@@ -4,15 +4,30 @@ hide_title: true
 sidebar_position: 2
 ---
 
+#### Version 3.0.2, 2.1.3
+
+| CVE            | Title                                                       |                   Affected |
+|:---------------|:------------------------------------------------------------|---------------------------:|
```
Comment (michael-s-molina) [inline docs/docs/security/cves.mdx:10]: @dpgaspar We can assume that all fixes in 2.1.3 are present in 3.0.2. I suggest moving these to the 2.1.3 block and keep the incremental version order.

Comment (dpgaspar) [reply docs/docs/security/cves.mdx:10]: could give the wrong impression that these fixes are included on 3.0.0 or 3.0.1

Comment (michael-s-molina) [reply docs/docs/security/cves.mdx:10]: You're right! Should we copy/duplicate them and keep the version order? It looks weird that 2.1.3 is defined twice.

**Review by michael-s-molina — APPROVED** [review]
LGTM. Left a non-blocking comment.

File: docs/docs/security/cves.mdx [hunk 2]
```diff
@@ -4,15 +4,30 @@ hide_title: true
...
-| CVE-2023-42504 | Lack of rate limiting allows for possible denial of service             |  < 3.0.0 |
 | CVE-2023-42505 | Sensitive information disclosure on db connection details               |  < 3.0.0 |
 
 
+#### Version 2.1.3
+
+| CVE            | Title                                                                   | Affected |
+|:---------------|:------------------------------------------------------------------------|---------:|
+| CVE-2023-42504 | Lack of rate limiting allows for possible denial of service             |  < 2.1.3 |
+
```
Comment (eschutho) [inline docs/docs/security/cves.mdx:29]: you can also add if needed
[CVE-2023-30608](https://github.com/advisories/GHSA-rrm6-wvj7-cwh2)
[CVE-2023-30861](https://github.com/advisories/GHSA-m2qf-hxjv-5gpq)

**Review by eschutho — APPROVED** [review]
LGTM.. left a comment but not a blocker.

File: docs/docs/security/cves.mdx (hunk 2 above)
Comment (dpgaspar) [reply docs/docs/security/cves.mdx:29]: I am not, we should track and patch these in an automated fashion

**Pull request merged**